Safety/Src/main.c

```diff
@@ -108,26 +107,23 @@ int main(void)
   MX_USART2_UART_Init();
 
   /* USER CODE BEGIN 2 */
-  PWM_Init();
 
   debug("\r\n\r\nStarting up...");
   debug("Compiled on %s at %s", __DATE__, __TIME__);
 
   HAL_GPIO_WritePin(LED1_GPIO_Port, LED1_Pin, GPIO_PIN_RESET);
   HAL_GPIO_WritePin(LED2_GPIO_Port, LED2_Pin, GPIO_PIN_SET);
   HAL_GPIO_WritePin(LED3_GPIO_Port, LED3_Pin, GPIO_PIN_SET);
```

Comment:
Need to use the LEDs for something useful.

Reply:
Out of scope for this PR, but yes we didn't spend all that money on LEDs for nothing :P

Reply:
I put an asana task for it, a good first year task probably